### PR TITLE
updates to riui for desktop launch

### PR DESCRIPTION
**Pull request by tropicadri**
With the launch of richer install ui this article needs updates.

### CODE REVIEW

File: site/en/blog/richer-pwa-installation/index.md [hunk 1]
```diff
@@ -80,12 +81,9 @@ and users' reaction.
 
 ## Previewing the UI {: #previewing }
 
-This UI works in M91 non stable channels (Chrome Dev and Chrome Canary) on Android, with the flag `#mobile-pwa-install-use-bottom-sheet` enabled in `chrome://flags`.
+This UI works in M94 on Android, and M108 on desktop.
```
Comment (beaufortfrancois) [inline site/en/blog/richer-pwa-installation/index.md:84]: Shall we link to your upcoming article for the `M108 on desktop` part?

File: site/en/blog/richer-pwa-installation/index.md [hunk 2]
```diff
@@ -74,118 +75,28 @@ alt="Richer Install UI Collapsed", width="342", height="722" %}
...
-the existing prompts. This may change in the future depending on uptake of the developer community 
-and users' reaction.  
+Websites that do not include at least one screenshot to their manifest file will continue to receive
+the existing prompts. This may change in the future depending on uptake of the developer community
+and users' reaction.
 
 ## Previewing the UI {: #previewing }
 
-This UI works in M91 non stable channels (Chrome Dev and Chrome Canary) on Android, with the flag `#mobile-pwa-install-use-bottom-sheet` enabled in `chrome://flags`.
+This UI works in M94 on Android, and [M108 on desktop](/blog/richer-install-ui-desktop/).
```
Comment (rachelandrew) [inline site/en/blog/richer-pwa-installation/index.md:84]:
```suggestion
This UI works from Chrome 94 on Android, and [Chrome 108 on desktop](/blog/richer-install-ui-desktop/).
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 3]
```diff
@@ -74,118 +75,28 @@ alt="Richer Install UI Collapsed", width="342", height="722" %}
...
-
-Descriptions are not compulsory, so there is no minimum requirement. However, there is a maximum that 
-kicks in after 7 lines of text (roughly 324 characters). After this an ellipse will appear and 
-truncate longer descriptions 
-([for example](https://glitch.com/edit/#!/richerinstall-longer-description)). 
-
-In the future we will consider adding other data such as categories and app rating, but this will 
-be based on feedback from developers and users. 
-[See the example code here](https://glitch.com/edit/#!/richerinstall-description?path=manifest.json%3A13%3A29).
+To display the richer install ui dialog developers need to add at least one screenshot for the corresponding form factor in the `screenshots` array. The `description` field is not required but it is recommended,  the content dialog is built with the content of those two fields and make the experience more similar to app store install, helping users identify they are adding an app to their device and with more space available developers can provide specific context to their users at install time.
```
Comment (rachelandrew) [inline site/en/blog/richer-pwa-installation/index.md:90]:
```suggestion
To display the richer install UI dialog developers need to add at least one screenshot for the corresponding form factor in the `screenshots` array. The `description` field is not required but it is recommended. The content dialog is built using the content of the `screenshots` and `description` fields to make the experience more similar to an app store install. This UI helps users identify they are adding an app to their device, and with more space available developers can provide specific context to their users at install time.
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 4]
```diff
@@ -43,7 +44,7 @@ somewhat confused about what has happened.
 To enable developers to provide installed experiences on par with native experiences
 Chrome is introducing a new install surface, Richer Install, that allows developers to add a
 description and screenshots to their manifest file and have it appear in a bottomsheet dialog
-within Chrome for Android. 
+within Chrome for Android.
 
 <figure>
   {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/RoQRxu3CPkKvAtkxYsAQ.jpeg", alt="Example of bottomsheet UI in Chrome", width="360", height="720" %}
```
Comment (rachelandrew) [inline site/en/blog/richer-pwa-installation/index.md:50]:
```suggestion
  {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/RoQRxu3CPkKvAtkxYsAQ.jpeg", alt="Example of bottomsheet UI in Chrome.", width="360", height="720" %}
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 5]
```diff
@@ -52,20 +53,20 @@ within Chrome for Android.
...
-of installed experiences. 
+of installed experiences.
 
 
 {% Columns %}
 {% Column %}
 <figure>
-  {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/SpStAtUk8Zp5iwi9yqKP.jpg", 
+  {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/SpStAtUk8Zp5iwi9yqKP.jpg",
 alt="Richer Install UI Expanded", width="342", height="722" %}
```
Comment (rachelandrew) [inline site/en/blog/richer-pwa-installation/index.md:63]:
```suggestion
alt="Richer Install UI Expanded.", width="342", height="722" %}
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 6]
```diff
@@ -52,20 +53,20 @@ within Chrome for Android.
...
   <figcaption>Richer Install UI Expanded.</figcaption>
 </figure>
 {% endColumn %}
 {% Column %}
 <figure>
-  {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/k7r4yKqrh6iOm2XyZHfw.jpg", 
+  {% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/k7r4yKqrh6iOm2XyZHfw.jpg",
 alt="Richer Install UI Collapsed", width="342", height="722" %}
```
Comment (rachelandrew) [inline site/en/blog/richer-pwa-installation/index.md:70]:
```suggestion
alt="Richer Install UI Collapsed.", width="342", height="722" %}
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**